Schutzfile

```diff
@@ -179,14 +179,14 @@
           {
             "title": "fedora",
             "name": "fedora",
-            "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/public/f40/f40-x86_64-rawhide-20240101"
```

Comment:
ugh we should just have a 'rawhide' branch until f40 comes out and then do f40. Instead of this -rawhide and -branched.

Reply:
I just copied what we did last time in rpmrepo.